src/screen.rs

```diff
@@ -547,6 +547,7 @@ impl Screen {
                 .map_or(1, |p| calc_prompt_lines(p));
             self.actual.cursor.y += prompt_line_count.checked_sub(1).unwrap();
             self.actual_left_prompt = None;
+            self.need_clear_screen = true;
```

Comment:
This seems fine but I guess always clearing to the end of screen is not always optimal:

If a background process does something like

	sh -c 'sleep .1 && printf "\033[5Becho hello\033[5A"' &

it seems better to not hide that (the user can always press `ctrl-l`).

So in future we should maybe only clear the lines we painted, or only clear if the prompt actually shrinked.